spec/features/admin/invoice_print_spec.rb

```diff
@@ -239,6 +239,119 @@
           expect(page).to have_content "Total (Excl. tax): $1,446.70"
         end
       end
+    end
+
+        context "added" do
```

Comment:
Whitespace! :wink:

Reply:
Thanks, incredible eye for detail :sparkles: 

This was corrected in the following commit :+1: